scarb/tests/package.rs

```diff
@@ -1636,3 +1640,90 @@ fn files_outside_package_cannot_be_included() {
             prefix not found
         "#});
 }
+
+#[test]
+fn package_script_is_run() {
+    let t = TempDir::new().unwrap();
+    simple_project()
+        .manifest_package_extra(indoc! {r#"
+            [scripts]
+            package = "echo 'Hello!'"
```

Comment:
In the original issue, it was mentioned that the `package` field specifies the path to a package script, not the script’s content itself. This still seems like the preferred approach to me.

A script is unlikely to be a one-liner as in these tests. Generating binaries for multiple platforms and naming them accordingly will likely require a few lines of code 😄. I’m not sure we want to keep all of this within the manifest. Additionally, having the script as a separate file would make it easier to dry-run the script. I don’t think we want to force users to run `scarb package` just for that.

Reply:
A path to a script is also a valid script (+/- the file is `chmod +x`ed which we can expect people to have anyway). This works as I expected.

<img width="1097" alt="image" src="https://github.com/user-attachments/assets/9bd3959c-41f5-4f57-bf4a-9f8b54063541" />

Reply:
It's handled by our scripts system, so there is no need to test it here, as the running of scripts is handled extensively in `tests/scripts.rs`